transformer/timestamp.go

```diff
@@ -61,6 +61,7 @@ func (config *Timestamp) Transform(c *skogul.Container) error {
 		if config.Format == "" {
 			config.Format = time.RFC3339
 		}
+		config.Format = parseTimestamp(config.Format)
```

Comment:
It's because we provide a convenience function to support short formats for standards (iso8601/RFC3339), but we need to set the actual format. It's a string constant from the time package, so we just set that. So it set the appropriate format if someone configured it by setting the format to "rfc3339" or "iso8601" instead of the full format string, as those are not valid formats.

Reply:
I can make it an un-exported field on the struct instead, yes. 😊

Reply:
I think that's marginally cleaner, since otherwise if we want to re-export the config, it will have changed. It's not like it wont change anyway, so this is probably a bit of a weird nitpick...

Reply:
Done 👍